### PR TITLE
Bluespace handcuffs

### PR DESCRIPTION
**Pull request by Jazz23**
**What does this PR do:**
If yall didn't already know I made a [shatload of new bs items](https://nanotrasen.se/forum/topic/15146-what-do-you-guys-wanna-be-bluespace/) all in one PR, and people requested I spread it out so here I am.

Introducing, the bluespace handcuffs. I never play sec so I have no idea why you'd use this but this person [this person requested it.](https://nanotrasen.se/forum/topic/15146-what-do-you-guys-wanna-be-bluespace/?tab=comments#comment-123537). ALL SPRITES WERE MADE BY @Ionward .

**Images of sprite/map changes (IF APPLICABLE):**
Sprites found [here](https://nanotrasen.se/forum/topic/15146-what-do-you-guys-wanna-be-bluespace).

**Changelog:**
:cl:
add: Bluespace Handcuffs
imageadd: Bluespace Handcuffs sprites.
/:cl:

### CODE REVIEW

**Comment by Bookdrizzt** [comment]
I would love this, especially if it goes in my security belts. It’d be like 3 handcuffs in one, and would vastly reduce the amount of cuffs I have to carry around because I need to be ready to cuff at a moment’s notice. 

Could you clarify the tech requirements for building this and making this, as well as the material costs?

File: code/game/objects/items/weapons/handcuffs.dm [hunk 1]
```diff
@@ -16,6 +16,9 @@
 	var/cuffsound = 'sound/weapons/handcuffs.ogg'
 	var/trashtype = null //For disposable cuffs
 	var/ignoresClumsy = FALSE
+	var/icon_hands_r = "markus" // icons/mob/screen_gen.dmi, its the actual icon of cuffs in your inventory when you are cuffed.
```
Comment (Citinited) [inline code/game/objects/items/weapons/handcuffs.dm]: What odd variable names (here and below)

File: .gitignore [hunk 2]
```diff
@@ -20,4 +20,7 @@ data/
 *.db
 stddef.dm
 .atom-build.json
-*.vscode/*
\ No newline at end of file
+*.vscode/*
```
Comment (Citinited) [inline .gitignore]: Are these lines meant to be added?

File: code/game/objects/items/weapons/handcuffs.dm [hunk 3]
```diff
@@ -194,3 +198,46 @@
...
+/obj/item/restraints/handcuffs/bluespace
+	name = "bluespace handcuffs"
+	desc = "A device that dispenses one-time energy cuffs to the victim."
+	icon_state = "handcuff_bluespace"
+	item_state = "handcuff_bluespace"
+	icon_overlay = "handcuff1_energy"
+	icon_hands_l = "energy_l"
+	icon_hands_r = "energy_r"
+	var/obj/item/stock_parts/cell/power_supply
+	var/can_charge = 1
```
Comment (Citinited) [inline code/game/objects/items/weapons/handcuffs.dm]: Are these three vars doing anything?

**Comment by variableundefined** [comment]
I love the sprites, however I don't think this handcuffs is a good idea. Once again, we do not need a bloat of "x but better" items, and this item would circumvent the compromise security members has to make between carrying non-lethal restraints and other gears.

**Pull request closed without merging**